test/BF16/mlp-single-layer-bf16.mlir

```diff
@@ -4,6 +4,10 @@
 // RUN: -shared-libs=%llvmlibdir/libmlir_c_runner_utils%shlibext,%tpplibdir/libtpp_c_runner_utils%shlibext
 //
 
+// Validate default pipeline
+// RUN: tpp-run %s -print \
+// RUN:  -e entry -entry-point-result=void
```

Comment:
These really should have CHECK lines on both, too. I know "it wasn't there before" but it would be easy to add and make sure that the test is really correct. We've been bitten by that recently...

Reply:
I reviewed this and other integration tests that don't use print once more. And I still don't see any point in adding CHECKs to these tests. Printing is not practical due to either buffer sizes e.g., ` memref<128x512xbf16>` in this test, or because they would just repeat existing validation using `check` ops.

Maybe I'm misinterpreting your goal here. But if there are any concerns about validation using `check` dialect, I'd rather add extra tests for `check` ops and keep using them whenever possible.

Reply:
Argh, sorry, check dialect checks. You're right, it's fine.